Quantum-Control-Applications/Superconducting/Single Fixed Transmon/configuration.py

```diff
@@ -211,16 +214,48 @@ def readout_macro(threshold=None, state=None, I=None, Q=None):
             "operation": "control",
             "length": x180_len,
             "waveforms": {
-                "I": "x180_drag_wf",
-                "Q": "x180_drag_der_wf",
+                "I": "x180_wf",
+                "Q": "x180_der_wf",
             },
         },
         "x90_pulse": {
```

Comment:
keep the order of operations the same as the pulses?
either 90 180 270 or 180 90 270